lib/dev/ui/spinner.rb

```diff
@@ -54,6 +54,7 @@ def check
             begin
               status = @thread.join.status
               @success = (status == false)
```

Comment:
No, nevermind, I'm thinking of some other block-runner thing in this codebase. I dunno.